packages/dataviews/src/filter-summary.js

```diff
@@ -24,43 +24,67 @@ import { ENTER, SPACE } from '@wordpress/keycodes';
  * Internal dependencies
  */
 import SearchWidget from './search-widget';
-import { OPERATOR_IN, OPERATOR_NOT_IN, OPERATORS } from './constants';
+import {
+	OPERATORS,
+	OPERATOR_IS,
+	OPERATOR_IS_NOT,
+	OPERATOR_IS_ANY,
+	OPERATOR_IS_NONE,
+} from './constants';
 
-const FilterText = ( { activeElement, filterInView, filter } ) => {
-	if ( activeElement === undefined ) {
+const FilterText = ( { activeElements, filterInView, filter } ) => {
+	if ( activeElements === undefined || activeElements.length === 0 ) {
 		return filter.name;
 	}
 
 	const filterTextWrappers = {
-		Span1: <span className="dataviews-filter-summary__filter-text-name" />,
-		Span2: <span className="dataviews-filter-summary__filter-text-value" />,
+		Name: <span className="dataviews-filter-summary__filter-text-name" />,
+		Value: <span className="dataviews-filter-summary__filter-text-value" />,
 	};
 
-	if (
-		activeElement !== undefined &&
-		filterInView?.operator === OPERATOR_IN
-	) {
+	if ( filterInView?.operator === OPERATOR_IS_ANY ) {
+		return createInterpolateElement(
+			sprintf(
+				/* translators: 1: Filter name. 3: Filter value. e.g.: "Author is any: Admin, Editor". */
+				__( '<Name>%1$s is any: </Name><Value>%2$s</Value>' ),
+				filter.name,
+				activeElements.map( ( element ) => element.label ).join( ', ' )
```

Comment:
I suspect that some (few) locales will want to have a say in the punctuation used to enumerate filters:

```js
activeElements.map( ... ).join(
  /* translators: string used to join together active filters, e.g. "Author is any: Admin, Editor" */
  __( ', ')
)
```

@swissspidy: do you know if this is done elsewhere in WP?